deployments/azure-container-webapp.yml

```diff
@@ -44,11 +44,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Setup Docker buildx
```

Comment:
Any reason to change the name? (is this the name that's also used to "mark check as required"? (if so, changing the name can be somewhat disruptive 😓)

Reply:
Ah right it's the other way around if we want to be consistent across our examples and docs https://docs.docker.com/build/ci/github-actions/